### ISSUE TITLE
 Background image replacement for Elementor not working when neve is active

### ISSUE THREAD

**Issue by selul** [Codeinwp/optimole-wp#542]
The issue still seems to occur in many cases, for example the user here face the [same issue](https://secure.helpscout.net/conversation/2236479199/368836?folderId=2353196).  The only workaround to fix this issue is by turning on the **Serve CSS/JS** option. On discussion with @GrigoreMihai, we came to the conclusion to reopen the issue with these findings.

_Originally posted by @kushh23 in https://github.com/Codeinwp/optimole-wp/issues/500#issuecomment-1550307269_

**Comment by selul** [comment]
@GrigoreMihai let me know your findings and why is still occuring.

**Comment by GrigoreMihai** [comment]
> @GrigoreMihai let me know your findings and why is still occuring.

This was still occuring because we didn't actually had any code change when it was first closed. 

Managed to track down what the issue was. When using some themes (neve,astra) the part where elementor uses `wp_get_attachment_image_url` was called earlier then the hook we used to add the filter. This was because it was going to a different css saving flow. 

I found this by editing elementor files and logging the `wp_debug_backtrace_summary`  inside the `get_style_value` function that was responsable for getting the src and then followed to the earliest filter/action I could find.
